### ISSUE TITLE
bring back terminal resizing by multiple lines

### ISSUE THREAD

**Issue by dein-ding** [microsoft/vscode#152381]
Previously the terminal could be resized fairly quickly, to see what's going on and then hide it again (with 2 lines still visible). With the new update however, it can _only_ be resized by one line per keystroke. So in order to use the terminal adequately, there are 10 - 20 keystrokes required, which is unacceptable (in my opinion). 

It would be great, if you could specify "args" for the command which control how many lines the terminal is resized by. Similar like the cursorMove command: 
```json
"command": "cursorMove",
"args": {
    "to": "down",
    "by": "line",
    "value": 10, <-- like this
},
```

Thanks for consideration, Floyd Haremsa

<!-- Describe the feature you'd like. -→

**Comment by meganrogge** [comment]
I'm not sure what you mean.
changing the height of the panel?
https://user-images.githubusercontent.com/29464607/174156563-43ea2c10-df58-4196-bb04-bfeace3e54da.mov

**Comment by dein-ding** [comment]
Yes, changing the height of the panel, but with keyboard shortcuts. Previously it worked pretty well (changing the height by lets say 10 lines, I don't quite remember). But now you can only resize by one line at a time (per hitting the keyboard shortcut), like this:

https://user-images.githubusercontent.com/84234327/174274405-5b3d5e52-31db-46e3-b7f1-a78a9c879a5b.mov

and that is just way too slow and inefficient. This way I'm forced to resize it with the mouse (which is also too slow) or having it open all the time, which is bad on small screen sizes.

**Comment by sbatten** [comment]
This is indeed caused by https://github.com/microsoft/vscode/commit/4309c752dde6e07c0c0ace2878cc8ff2d44622e1. The new behavior is actually the intended behavior, but the terminal is in full control if it wants to have a number-of-lines-based resize setting.

https://github.com/microsoft/vscode/blob/f17b33faf21feba942e4663e40226456b081ae5d/src/vs/workbench/contrib/terminal/browser/terminalGroup.ts#L64

**Comment by vlrevolution** [comment]
I am also affected by this and it is crazy to work mouseless, resizing terminal is not viable with keyboard like this. Any work around?

**Comment by Tyriar** [comment]
@dein-ding @vlrevolution how many rows would you expect it to change?

**Comment by vlrevolution** [comment]
> @dein-ding @vlrevolution how many rows would you expect it to change?

3-4 would be nice

**Comment by dein-ding** [comment]
> @dein-ding @vlrevolution how many rows would you expect it to change?

I'd say about 7 - 10 lines at a time. But if I can control it my self, that's even better.

**Comment by muendelezaji** [comment]
Also being affected by this. The terminal now resizes by 1 line at a time. Previously it was 9 lines (3072×1920 on 16" with terminal.integrated.fontSize = 13)

> @dein-ding @vlrevolution how many rows would you expect it to change?

Something in the range of 4-8 lines might be a decent compromise considering the original request that led to the change (#149488)

**Comment by dein-ding** [comment]
I think the best approach would be to just have a default of let's say 4 lines, but also a setting that controls this, and/or as I mentioned, arguments to the command for being able to map multiple resize settings to different shortcuts:
```JSON
"command": "cursorMove",
"args": {
    "to": "down",
    "by": "line",
    "value": 10, <-- like this
},
```

**Comment by Tyriar** [comment]
I'd rather not add a setting for this as it's pretty obscure and would see hardly any usage. An additional complication here is I'd like to eventually remove the terminal resize commands since we now have the increase/decrease view size commands which weren't around when the terminal resize pane commands got added. Seems like 4 rows should work for most people.

**Comment by muendelezaji** [comment]
Can confirm this is fixed in Insiders. Thanks for the quick response
